app/jobs/nuntius/messenger_job.rb

```diff
@@ -21,8 +21,8 @@ def perform(obj, event, params = {})
     def cleanup_nuntius_events(obj, event)
 
       nuntius_events = Nuntius::Event.where(
```

Comment:
Let's make this delete_all, so `Nuntius::Event.where(...).delete_all` - should be faster too. 
There's no need to instantiate these and then call destroy, right?